internal/service/encryptionatrest/resource_migration_test.go

```diff
@@ -37,7 +38,7 @@ func TestMigEncryptionAtRest_basicAWS(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				ExternalProviders: mig.ExternalProviders(),
-				Config:            testAccMongoDBAtlasEncryptionAtRestConfigAwsKms(projectID, &awsKms, false), // not using data source as it was introduced in 1.19.0
+				Config:            testAccMongoDBAtlasEncryptionAtRestConfigAwsKms(projectID, &awsKms, useDatasource), // not using data source as it was introduced in 1.19.0
```

Comment:
knit: you may want to do some opportunistic improvements, like renaming `testAccMongoDBAtlasEncryptionAtRestConfigAwsKms` to `configAwsKms', etc.

Reply:
updated, thanks for the tip!